clustermesh-apiserver/main.go

```diff
@@ -543,6 +547,7 @@ func runServer(cmd *cobra.Command) {
 		if err := k8s.Init(k8sconfig.NewDefaultConfiguration()); err != nil {
 			log.WithError(err).Fatal("Unable to connect to Kubernetes apiserver")
 		}
+		synced.WaitForCRDsToRegister(context.TODO(), &synced.Resources{}, &synced.APIGroups{})
```

Comment:
Is the `CRDWaitTimeout` supposed to be used to create a context and passed in here?

Reply:
I'm also not entirely clear why we need to also declare the CRDWaitTimeout flag again in this file.

Reply:
Seems not needed, as there is a fatal log upon CRDWaitTimeout within the waiter code. Bumped into it before I added the code to use the `CRDWaitTimeout` option.